### PR TITLE
Add some translations - Chinese

### PR DESCRIPTION
**Pull request by tengqm**
This PR also includes some translations for glossary which have to be
included to make gate happy.

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.13 Features: set Milestone to 1.13 and Base Branch to dev-1.13
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by chenrui333** [comment]
I guess, we have already fixed in ##11310 and #11311.

**Comment by chenrui333** [comment]
If possible, we can have you join the i18n chinese localization group.
https://github.com/k8smeetup/k8s-official-translation

**Comment by markthink** [comment]
@tengqm 我已经向您发邀请了

File: content/zh/docs/reference/glossary/managed-service.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
 ---
+title: 托管服务
+id: managed-service
+date: 2018-04-12
+full_link: 
+short_description: >
+  由第三方供应商负责维护的一种软件交付件。
```
Comment (chenrui333) [inline content/zh/docs/reference/glossary/managed-service.md]: `软件交付件` or `交付软件`？

File: content/zh/docs/reference/glossary/managed-service.md [hunk 2]
```diff
@@ -9,9 +22,21 @@ short_description: >
...
 tags:
 - extension
----
+-->
+
+<!--
  A software offering maintained by a third-party provider.
+-->
+
+由第三方供应商负责维护的一种软件交付件。
```
Comment (chenrui333) [inline content/zh/docs/reference/glossary/managed-service.md]: same as above.

File: content/zh/docs/reference/glossary/managed-service.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/reference/glossary/managed-service.md]: 软件交付件确实生硬，搜索了一下，用的比较多的翻译是“软件产品”。

**Pull request merged**